README.md

```diff
@@ -1,26 +1,25 @@
 # ETW (Event Tracing for Windows)
```

Comment:
I would add a change-log for this release since we have many internal changes to the library.

Reply:
Good idea.

Reply:
Done.